### DISCUSSION TITLE
Advancement Criteria

### DISCUSSION THREAD

**Discussion by EpicPlayerA10** [Minestom/Minestom#1017] [Help] [answered]
Is there a way to add more than one criteria to advancement? For now in advancement there is one hardcoded criteria.

**Comment by LooFifteen** [comment, marked as answer]
[Protocol suggests that multiple criteria are possible](https://wiki.vg/Protocol#Advancements) but [it doesn't seem like it's possible in Minestom yet](https://github.com/Minestom/Minestom/blob/master/src/main/java/net/minestom/server/advancements/Advancement.java#L43).

**Reply by EpicPlayerA10** [reply]
Thanks, I will do a PR soon for multiple criteria.
